pkg/controllermanager/apis/config/v1alpha1/defaults.go

```diff
@@ -39,6 +39,14 @@ func SetDefaults_ControllerManagerConfiguration(obj *ControllerManagerConfigurat
 		obj.Controllers.Bastion.MaxLifetime = &metav1.Duration{Duration: 24 * time.Hour}
 	}
 
+	if obj.Controllers.CertificateSigningRequest == nil {
+		obj.Controllers.CertificateSigningRequest = &CertificateSigningRequestControllerConfiguration{}
+	}
+	if obj.Controllers.CertificateSigningRequest.ConcurrentSyncs == nil {
+		v := DefaultControllerConcurrentSyncs
+		obj.Controllers.CertificateSigningRequest.ConcurrentSyncs = &v
+	}
+
```

Comment:
Can you put this into a dedicated `SetDefaults_CertificateSigningRequestControllerConfiguration` function, please?